source/includes/steps-monitor-with-snmp.yaml

```diff
@@ -1,32 +1,177 @@
 title: Copy configuration files.
 stepnum: 1
 ref: copy-config-files
-pre: |
-  Use the following sequence of commands to move the SNMP
-  configuration files to the SNMP service configuration directory.
+content: |
+  Copy the MongoDB SNMP configuration files from their installation
+  location to their target directories, according to your platform:
+  
+  .. tabs::
```

Comment:
Heh -- this is actually included in Monitor MongoDB **Windows** with SNMP page

https://docs-mongodbcom-staging.corp.mongodb.com/andrew.feierabend/DOCS-5839-clarify-snmp-trap-and-bind/tutorial/monitor-with-snmp-on-windows.html

Reply:
You're right. Fixed by moving the Windows-specific copy to its own file.